.yamato/config/__shared.metafile

```diff
@@ -83,11 +90,18 @@ test_platforms:
       - [Android_OpenGLES3, Android_Vulkan]: --timeout=2700
       - [iPhone]: --timeout=1800
       
-      - [Linux_OpenGlCore]: --extra-editor-arg="-executemethod" --extra-editor-arg="CustomBuild.BuildLinuxOpenGlCoreLinear"
-      - [Linux_Vulkan]: --extra-editor-arg="-executemethod" --extra-editor-arg="CustomBuild.BuildLinuxVulkanLinear"
-      - [Win_DX11]: --extra-editor-arg="-executemethod" --extra-editor-arg="CustomBuild.BuildWindowsDX11Linear"
-      - [Win_DX12]: --extra-editor-arg="-executemethod" --extra-editor-arg="CustomBuild.BuildWindowsDX12Linear"
-      - [Win_Vulkan]: --extra-editor-arg="-executemethod" --extra-editor-arg="CustomBuild.BuildWindowsVulkanLinear"
+      #- [Linux_OpenGlCore]: --extra-editor-arg="-executemethod" --extra-editor-arg="CustomBuild.BuildLinuxOpenGlCoreLinear"
```

Comment:
Genuine question as I'm not totally familiar with the new args system yet, why these were commented out?

Reply:
answered in a response to Liis